src/intl/en/learn-quizzes.json

```
@@ -613,5 +613,50 @@
+  "smart-contracts-1-d-label": "They are the rules behind the Ethereum Blockchain, developed together with lawyers to ensure legal compliance.",
+  "smart-contracts-1-d-explanation": "Smart contracts are pieces of code that can be created by developers and deployed on a blockchain.",
+  "smart-contracts-2-prompt": "What is a metaphor that most closely characterizes the functioning of smart contracts?",
+  "smart-contracts-2-a-label": "A bank",
+  "smart-contracts-2-a-explanation": "Banks require manual intervention and are structured as hierarchical entities, whereas a smart contract automatically executes without intervention.",
+  "smart-contracts-2-b-label": "A digital vending machine",
+  "smart-contracts-2-b-explanation": "The vending machine will only dispense your desired product after all requirements are met - specific inputs guarantee predetermined outputs. This is similar to the logic of smart contracts.",
+  "smart-contracts-2-c-label": "A calculator",
+  "smart-contracts-2-c-explanation": "smart contract code can be used for calculations, but is not limited to that. Rather, smart contracts are blockchain-based programs that follow an 'if this then that' logic.",
+  "smart-contracts-2-d-label": "A website",
+  "smart-contracts-2-d-explanation": "A website is the frontend which captures user directions. A smart contract is the backend logic where these directions are executed, and the result may be returned.",
+  "smart-contracts-3-prompt": "What is not one of the main characteristics of a smart contract?",
+  "smart-contracts-3-a-label": "Automatic execution",
+  "smart-contracts-3-a-explanation": "The main benefit of a smart contract is that it deterministically executes unambiguous code when certain conditions are met.",
+  "smart-contracts-3-b-label": "Public record",
+  "smart-contracts-3-b-explanation": "With smart contracts on a public blockchain, anyone can instantly track asset transfers and other related information.",
+  "smart-contracts-3-c-label": "Privacy protection",
+  "smart-contracts-3-c-explanation": "As blockchains are pseudonymous networks, transactions are tied publicly to a unique cryptographic address, not an identity.",
+  "smart-contracts-3-d-label": "Ability to change",
+  "smart-contracts-3-d-explanation": "A smart contract cannot be changed once created, it is guaranteed to execute according to the rules defined by its code.",
+  "smart-contracts-4-prompt": "What is not one of the applications of smart contracts?",
+  "smart-contracts-4-a-label": "Cryptocurrencies",
+  "smart-contracts-4-a-explanation": "Cryptocurrencies are token objects which are defined and tracked on smart contracts.",
+  "smart-contracts-4-b-label": "Protocol changes",
+  "smart-contracts-4-b-explanation": "While protocol changes are enacted through a smart contract, their creation and definition are proposed through transparent online forums such as Github or Forum.",
+  "smart-contracts-4-c-label": "Non Fungible Assets",
```

```suggestion
  "smart-contracts-4-c-label": "Non-fungible assets",
```

---

Or, since the explanation here used "NFT" consider just using "Non-fungible tokens"

---

Yeah, would consider spelling out and showing abbreviation for clarity, same with DAO below:

```suggestion
  "smart-contracts-4-c-label": "Non-fungible tokens (NFTs)",
```